tools/missing-test-detector/main.go

```diff
@@ -23,11 +24,15 @@ func main() {
 	if err != nil {
 		glog.Errorf("error detecting missing tests: %v", err)
 	}
-	for resourceName, missingTestInfo := range missingTests {
-		fmt.Printf("Resource %s changed\n", resourceName)
-		glog.Infof("Tests parsed: %v", missingTestInfo.Tests)
-		if len(missingTestInfo.UntestedFields) > 0 {
-			fmt.Printf("Untested fields: %v\n", missingTestInfo.UntestedFields)
+	if len(missingTests) > 0 {
+		fmt.Println("## Missing test report\nYour PR includes resource fields which are not covered by any test.")
```

Comment:
Could you switch the output to render a Go template instead of printing each line? That would make it easier to read & edit (and easier to include the per-field backticks in this part of the logic)